src/getDefaultSequencerInboxMaxTimeVariation.ts

```diff
@@ -12,17 +13,25 @@ export type SequencerInboxMaxTimeVariation = {
 export function getDefaultSequencerInboxMaxTimeVariation(
   parentChainIdOrPublicClient: ParentChainId | PublicClient,
 ): SequencerInboxMaxTimeVariation {
-  validateParentChain(parentChainIdOrPublicClient);
+  const parentChainId = validateParentChain(parentChainIdOrPublicClient);
 
   const delaySeconds = 60 * 60 * 24 * 4; // 4 days;
-  const delayBlocks = delaySeconds / 12;
-
   const futureSeconds = 60 * 60; // 1 hour;
-  const futureBlocks = futureSeconds / 12;
+
+  if (parentChainId === base.id || parentChainId === baseSepolia.id) {
+    // For Arbitrum L3s built on top of an OP Stack L2, `block.number` will return the L2 block number.
```

Comment:
```suggestion
    // For Orbit L3s built on top of an OP Stack L2, `block.number` will return the L2 block number.
```
Not sure whether to use Orbit or Arbitrum here (not my call either, lol)

Reply:
(If you apply this change, the same comment appears in getDefaultConfirmPeriodBlocks.